wowchemy/layouts/slides/baseof.html

```diff
@@ -20,55 +21,48 @@
 
   <title>{{ .Title }} | {{ site.Title }}</title>
 
-  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/css/reveal.min.css">
+  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/reveal.min.css">
   {{- $theme := $.Param "slides.theme" | default "black" -}}
-  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/css/theme/{{ $theme }}.min.css">
+  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/theme/{{ $theme }}.min.css" />
 
+  {{ printf "<script src=\"%s\" integrity=\"%s\" crossorigin=\"anonymous\"></script>" (printf $js.jQuery.url $js.jQuery.version) $js.jQuery.sri | safeHTML }}
```

Comment:
jQuery appears to be redundant now that `fixMermaid` is vanilla JS?

Reply:
yes, I cannot remove since it will break slides without receiving the updated refactor.

don't forget this part as well:

```js
// document ready
  $(document).ready(function () {
    fixMermaid();
  });
```

Reply:
im assuming u will take care of this when merging the 2 PRs